packages/document/docs/en/api/config/config-frontmatter.mdx

```diff
@@ -245,3 +245,9 @@ Or
 overviewHeaders: [2, 3]
 ---
 ```
+
+## context
+
+- Type: `string`
+
+After configuration, the `data-context` attribute will be added to the DOM node when the sidebar is generated, and the value is the configured value.
```

Comment:
Can we add the example from https://github.com/web-infra-dev/rspress/issues/1158 to our document?

This will make the usage more clear.

Reply:
added